src/functionalTest/resources/sourceFiles/service-test-failure.csv

```diff
@@ -0,0 +1,2 @@
+service_code,ccd_service_name,ccd_case_type
+AAA2,ccd-service2,""
```

Comment:
why will this fail??

Reply:
file wont have double quotes in the end.

Reply:
have a look updated file.

Reply:
i dont see it as updated...

Reply:
And my previous question was why is this scenario expected to fail?
I think it is a valid scenario and should pass and not fail

Reply:
We are failing in case file does not have any valid record. If all records are invalid.